tst/CTA.Rules.Test/CTAFiles/project.specific.json

```diff
@@ -1,5 +1,41 @@
 {
   "NameSpaces": [
+    {
+      "namespace": "Project",
+      "Assembly": "Project",
+      "Reference": "Project",
+      "Type": "Project",
+      "Actions": [
+        {
+          "Name": "CreateMonolithService",
+          "Type": "Project",
+          "Value": {
+            "namespaceString": "MvcMusicStore.Controllers",
+            "projectName": "MvcMusicStore"
+          },
+          "Description": "Create service class."
+        }
+      ]
+    },
+    {
+      "Namespace": "MvcMusicStore.Controllers",
+      "Assembly": "MvcMusicStore",
+      "Classes": [
+        {
+          "Key": "StoreManagerController",
+          "FullKey": "MvcMusicStore.Controllers.StoreManagerController",
+          "KeyType": "ClassName",
+          "Actions": [
+            {
+              "Name": "ReplaceMvcControllerMethodsBody",
+              "Type": "Class",
+              "Value": "return Content(MonolithService.CreateRequest(Request, this.ControllerContext.RouteData));",
+              "Description": "Replace method body with http request to extracted service."
```

Comment:
Can we add a test for Web API?

Reply:
Agreed, added one for Action UT and one for whole solution UT.